.vscode/settings.json

```diff
@@ -7,5 +7,11 @@
     "editor.formatOnSave": true,
     "editor.tabSize": 2,
 
-  }
+  },
+  "favorites.resources": [
```

Comment:
Ah, it is for an extension I have that allows you to favourite files/folders in vscode and pin them on the sidebar. This shouldn't really be here but hard to avoid including these changes if we want to continue to commit `settings.json`.